aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,58 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 4, 2022 - April 8, 2022
+
+### Latest Releases
+
+- **grommet-theme-hpe v3.0.0** - This version ships with additional responsive breakpoints to help deliver the optimal user experience for HPE customers.
+  - Gain precise control over how page layouts respond and adapt.
+  - Implement grid layouts, such as a dashboard of cards, which gracefully adapt and flow.
+  - These additional breakpoints are breaking changes from v2 that will require some minor code changes to ensure backwards compatibility. A [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-2.x-to-3.x) is available to make this a smooth process.
+
+## March 9, 2022 - April 1, 2022
+
+### Communications
+
+- We are actively working on a process for receiving design contributions from members outside of the Design System core team.
+- We will begin this process by rolling out a way for designers to contribute small fixes and medium enhancements by leveraging the Figma Branching feature. Stay tuned!
+
+
+### Roadmap Updates
+
+- **Page Layouts**
+  - Updated the [Page Layouts Figma file](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2891%3A21273) to reflect new Page component in Grommet. Page component guidance and examples coming soon to the Design System site.
+- **Global Banner Notifications**
+  - Guidance and examples are now available on the [Design System site](/templates/global-banner-notifications).
+  - Theme-related styling is available on `grommet-theme-hpe v2.4.0`.
+- **Page Header**
+  - Finalized gathering use cases, annotating designs from service teams, and exploring anatomy. [Design ideations](https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header) are in progress.
+  - Conducted a user study test on contextual navigation with regards to the "Ascending" Button. The goal was to understand user expectations when navigating up a navigation hierarchy and to determine the most intuitive name for this navigational component. Design decisions and guidance will be coming soon.
+- **Tabs**
+  - Work is in progress to provide design and development guidance for how Tabs should display when there is not ample screen width to fit all available tabs.
+  - Gathered use cases from stakeholders to understand scenarios when more than 5 tabs may be necessary on a single page.
+  - Design and best practice ideation is in progress.
+
+
+### Stay Current with the Latest Changes
+- Enhanced Grid documentation to help developers better understand [Grid's](/components/grid#grid-properties-and-key-concepts) capabilities and how to master this component to create dynamic layouts.
+- Added an example of a Form with a [single required FormField](https://design-system.hpe.design/templates/forms#single-required-formfield) on the site.
+- FileInput - Trash icon got replaced with a [Close icon](/components/fileinput). 
```

Comment:
```suggestion
- FileInput - For [insert the reason this change was made] the [remove file action](/components/fileinput) is now represented by the Close icon instead of the Trash icon. This change has been made in the Figma component and is available in [insert Grommet / grommet-hpe-theme version].
```